pkgs/tools/networking/tinystatus/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, stdenvNoCC
+, makeWrapper
+, netcat
+, curl
+, unixtools
+, coreutils
+, mktemp
+, findutils
+, gnugrep
+, fetchFromGitHub
+,
+}:
+stdenvNoCC.mkDerivation rec {
+  postFetch = ''
+    mkdir -p $out/bin
+    tar xf $downloadedFile --strip 1
+    patchShebangs tinystatus
+    mv tinystatus $out/bin/tinystatus
+  '';
+  buildInputs = [ makeWrapper ];
+  pname = "tinystatus";
+  version = "unstable-2021-07-09";
+  src = fetchFromGitHub {
+    owner = "bderenzo";
+    repo = "tinystatus";
+    rev = "fc128adf240261ac99ea3e3be8d65a92eda52a73";
+    sha256 = "sha256-FvQwibm6F10l9/U3RnNTGu+C2JjHOwbv62VxXAfI7/s=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 tinystatus $out/bin/tinystatus
+    wrapProgram $out/bin/tinystatus \
+      --set PATH "${lib.makeBinPath [stdenvNoCC curl netcat unixtools.ping coreutils mktemp findutils gnugrep]}"
+    runHook postInstall
+  '';
+  meta = with lib; {
+    description = "A static HTML status page generator written in pure shell.";
```

```suggestion
    description = "A static HTML status page generator written in pure shell";
```

contributing requires trailing no periods

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md?plain=1#L40-L43